run.sh

```diff
@@ -1,15 +1,8 @@
 #!/bin/bash
 
-set -e
-
-REDIS_HOST=${REDIS_HOST:-localhost}
-REDIS_HOST=${REDIS_HOST:-127.0.0.1}
-REDIS_PORT=${REDIS_PORT:-6379}
-SITEROOT=${SITEROOT:-/rotm}
-GA_TAG_ID=${GA_TAG_ID}
-echo "starting the service"
-
-cp -r /app/public/* /public/
-SECURE_PROTOCOL=true npm start
-
+if [ "$NODE_ENV" = "ci" ]
+then echo "starting service"
+  SITEROOT=/rtm
+fi
 
+exec npm start
```

Comment:
@JoeChapman I've updated the Dockerfile to run the script. But I can't answer your question on what SITEROOT is meant to do